src/main/resources/alma/maps/deweyLabels.tsv

```diff
@@ -0,0 +1,45 @@
+004.62	Kopplungs- und Kommunikationsprotokolle
```

Comment:
don't put it into the repo, for when copy the "other file" to this location this other file would be overriden everytime we pull.

Reply:
As I said at the moment we do not have any deweyLabels.tsv in the maps folder. Either we use this here or you have to create a new file on the server.

Reply:
" we do not have any deweyLabels.tsv in the maps folder"
but we have - in the test-folder. The files there are also available when calling them from "src".
See add11eeaf56eac75cae0c9c230c8b40b2750a556.